src/components/heading.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+class heading extends Component {
+  constructor() {
+    super();
+    this.state = { add: "" };
+    // this.event = this.event.bind(this);
```

Comment:
you should remove commented out code.

Reply:
@theAndrewCline ok can I do that after completing this app, I just want to make sure where I came from for practice purpose.